ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/SupportsMempool.hs

```diff
@@ -185,6 +185,26 @@ class ( UpdateLedger blk
   -- transaction size.
   getTransactionKeySets :: GenTx blk -> LedgerTables (LedgerState blk) KeysMK
 
+  -- Mempools live in a single slot so in the hard fork block case
```

Comment:
Could you explicitly point out here the caveat we had talked through the other day: that the cost savings here is non-zero, but relatively small compared to the eg network costs of mempool synchronization? Unlike PR https://github.com/IntersectMBO/ouroboros-consensus/pull/1382, in particular. As as of PR 1382, this code is no longer on the critical path of block (minting and) propagation.

Reply:
Done